storage/ndb/rest-server/rest-api-server/internal/feature_store/avro_parser.go

```diff
@@ -0,0 +1,215 @@
+/*
+ * This file is part of the RonDB REST API Server
+ * Copyright (c) 2023 Hopsworks AB
```

Comment:
2023, 2025

Reply:
This comment applies to all files that have a copyright header, not only the new files